addon/components/polaris-labelled.js

```diff
@@ -69,9 +69,9 @@ export default class PolarisLabelled extends Component {
 
     return guidFor(this);
   }
+
   set id(value) {
     this._id = value;
-    return value;
```

Comment:
just double-checking, we don't need to explicitly return?

Reply:
No, setter values are ignored by the JS runtime(the [eslint rule docs](https://eslint.org/docs/rules/no-setter-return) has more info on this)